docs/2101UserGuide.md

```diff
@@ -321,69 +324,104 @@ List of commands that can be undone:
 
 #### Redoing a command: `redo`
 
-Redoes a command that the user has undone previously.
+Accidentally used `undo`? Don’t worry, you can redo your undone commands to retrieve your previous data too!
 
-Format: `redo`
+Here’s how you can use it: `redo`
 
-<hr class="command-separator">
-
-#### Clear all Members and Events: `clear`
-Clears all Member and Event entries from CCACommander.
+Here's a list of commands you can redo:
+* `clear`
+* `createMember`
+* `deleteMember`
+* `editMember`
+* `createEvent`
+* `deleteEvent`
+* `editEvent`
+* `enrol`
+* `unenrol`
+* `editEnrolment`
```

Comment:
I think the list of commands here is not applicable to the `redo` command, since you can only redo a command that was undone. In other words, you cannot specifically redo a `clear` or `enrol` command, you can only redo an `undo` command.

Reply:
good point I'll remove this list